src/mono/wasm/runtime/crypto-worker.ts

```diff
@@ -41,7 +41,7 @@ export function init_crypto(): void {
         console.debug("MONO_WASM: Initializing Crypto WebWorker");
 
         const chan = LibraryChannel.create(1024); // 1024 is the buffer size in char units.
-        const worker = new Worker("dotnet-crypto-worker.js");
+        const worker = new Worker(`${Module.baseUrl}dotnet-crypto-worker.js`);
```

Comment:
if Module.baseUrl is undefined this would cause path resolution to fail in a strange way by inserting 'undefined' at the front, it would be better to default it to '' or check and throw. In both cases the situation is worse because errors in web workers are typically swallowed (it's possible an invalid worker path would throw in the browser tab, at least)

Reply:
nvm, I see you take steps to make sure it's initialized, I missed it